packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -1133,6 +1133,23 @@ describe('NetworkController', () => {
     });
   });
 
+  describe('refreshNetworkMetadataByNetworkClientId', () => {
+    it('updates the network status', async () => {
```

Comment:
This seems to just test a portion of the happy path. Can we test more of the behavior?